lib/flutter_libphonenumber_web.dart

```diff
@@ -0,0 +1,121 @@
+import 'dart:ui';
+
+import 'package:flutter/services.dart';
+import 'package:flutter_libphonenumber/src/web/base.dart';
+import 'package:flutter_libphonenumber/src/web/libphonenumber.dart';
+import 'package:flutter_libphonenumber/src/web/utils.dart';
+import 'package:flutter_web_plugins/flutter_web_plugins.dart';
+
+import 'dart:html' as html;
+
+const String libPhoneNumberVersion = '3.2.32';
+const String libPhoneNumberUrl =
+    'https://cdn.jsdelivr.net/npm/google-libphonenumber@$libPhoneNumberVersion/dist/libphonenumber.min.js';
```

Comment:
My only feedback is if we can lock this to a git commit instead of npm version.

```
const String libPhoneNumberVersion = 'b7fe84af9b553f0f2db765a6e20c27fa867a971d';
const String libPhoneNumberUrl = 'https://cdn.jsdelivr.net/gh/ruimarinho/google-libphonenumber@$libPhoneNumberVersion/dist/libphonenumber.min.js';
```